src/App.js

```diff
@@ -1,17 +1,68 @@
 import './App.scss';
 
-import React, { useEffect } from 'react';
+import React, { useEffect, createContext, useState } from 'react';
 
 import PropTypes from 'prop-types';
 import { Routes } from './Routes';
 
+import PageLoading from './PresentationalComponents/PageLoading/PageLoading';
+
+export const PermissionContext = createContext();
+
 const App = (props) => {
+
+    const [permissions, setPermissions] = useState({
+        customPolicies: false,
+        compliance: false,
+        drift: false,
+        insights: false,
```

Comment:
wait this is still insights? shouldn't it be advisor ? (maybe insights is technically correct, butttttttt) 🤔

Reply:
I can change it to advisor, I wasn't sure since the actual permission is `insights:*:*`

Reply:
ah ok, i wasn't sure what the permission was,  looks like yah gotta change it in this const

Reply:
Yeah I'll change it